.travis.yml

```diff
@@ -1,19 +1,19 @@
-sudo: required
-
-services:
- - docker
-
-before_install:
- - echo $PWD
- - docker pull dalg24/vera_tpls
-
-script:
- - docker run -v $PWD:/mnt/Futility dalg24/vera_tpls /bin/bash -c "mkdir /build; cd /build; cmake /mnt/Futility; make -j2 |& tee /mnt/Futility/make.out; ctest -E DA32 -j2"
- - ls
- - (exit `grep -i -B 4 'Unused variable' make.out | grep 'f90' | wc -l`)
- - git clone https://github.com/trilinos/Trilinos.git
- - docker run -v $PWD:/mnt/Futility dalg24/vera_tpls /bin/bash -c "mkdir /build; cd /build; /bin/bash /mnt/Futility/build_scripts/build_on_docker.sh /mnt/Futility; make -j2; ctest -E DA32 -j2"
-
-branches:
-  only:
-    - master
+sudo: required
+
+services:
+ - docker
+
+before_install:
+ - echo $PWD
+ - docker pull dalg24/vera_tpls
+
+script:
+ - docker run -v $PWD:/mnt/Futility dalg24/vera_tpls /bin/bash -c "mkdir /build; cd /build; cmake /mnt/Futility; make -j2 |& tee /mnt/Futility/make.out; ctest -E DA32 -j2"
```

Comment:
I did miss one. The reason the whole file looks changed is the dos2unix. I did only remove the `-E DA32` option from (now) both instances in the .travis.yml

If you click the individual commits here it'll show you what changed more specifically (in the one commit) rather than just diffed with master.